misc.go

```diff
@@ -199,6 +199,22 @@ func postSlackMethod(ctx context.Context, client HTTPRequester, path string, val
 	return postForm(ctx, client, SLACK_API+path, values, intf, debug)
 }
 
+// get a slack web method.
+func getSlackMethod(ctx context.Context, client HTTPRequester, path string, values url.Values, intf interface{}, debug bool) error {
+	return getResource(ctx, client, SLACK_API+path, values, intf, debug)
+}
+
+func getResource(ctx context.Context, client HTTPRequester, endpoint string, values url.Values, intf interface{}, debug bool) error {
```

Comment:
mind doing some minor renaming while you're here?

rename `doPost` to `doRequest` and rename this method to `getRequest`

Reply:
This should now be complete.